### PR TITLE
feat: Show a warning notification when an user enters with a non-recommended browser

### PR DESCRIPTION
**Pull request by sandrade-dcl**
## What?
When an user enters the explorer with a non-recommended browser, the landing page will show the next warning:

**_Your browser is not among the recommended choices for an optimal experience in Decentraland. We suggest you use one based on Chromium or Firefox._**

This warning message is for information purposes only. The user will always be able to enter the world even if he is running with a non-recommended (but compatible) browser.

**Recommended browsers:**
- Any browser based on Chromium (Chrome, Brave, Edge, etc.).
- Firefox.

## Implementation
We have added the `isRecommendedBrowser()` function in the `utils` module. This function will let us know when an user is using a recommended browser or not.

The function is used in `LoginConatainer.tsx` in order to show a warning message when an user use a non-recommended browser:

#### User entering with a recommended browser
image

#### User entering with a non-recommended browser
image

## Test Link
https://explorer-artifacts.decentraland.org/@dcl/explorer-website/branch/feat/Add-isRecommendedBrowser-function-to-website-utils/?kernel-branch=main&renderer-branch=master

### CODE REVIEW

**Comment by menduz** [comment]
I'll keep this on hold until we have the final implementation to not add "dead code" to the main branch, please continue the implementation of the errors in this same PR when necessary

**Comment by RominaMarchetti** [comment]
The new design for this kind of notifications should look like this from now (ignore the Play website re-desing for the moment): https://www.figma.com/file/gJHiKqv0E5YUz5so1CQlYD/SignIn-V2-and-Launcher?node-id=1836%3A29

Resume: Twhite text over dark background plus a thin gradient line at the bottom. Chrome and Firefox are links to their pages. The close button quit the notification during that session. This means that returning users should see the message every time they come back using a not recommended browser. 

@2fd can you give Santi a hand here?

File: src/utils/browser.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+export const recommendedBrowsers: string[] = [
+    "Chrome",
+    "Firefox"
+  ]
+  
+  export function isRecommendedBrowser() {
+    for (let i = 0; i < recommendedBrowsers.length; i++) {
```
Comment (LautaroPetaccio) [inline src/utils/browser.ts:7]: I think that changing this for
```
return recommendedBrowsers.some((browser) => navigator.userAgent.indexOf(browser) !== -1)
```
Might be cleaner. What do you think?

**Review by jmoguilevsky — APPROVED** [review]

**Review by LautaroPetaccio — CHANGES_REQUESTED** [review]
There are some minor style changes, but there's one which I think that we should fix, the dialog padding between "Decentraland" and the "Play" button.

Screen Shot 2021-09-01 at 12 15 01
Screen Shot 2021-09-01 at 12 14 57

This should be easily fixable by using `margin-top` and `margin-bottom` with a margin similar of the one we have in production in the class that was added to color that text.

The text in general is slightly darker, as it's not the same one that you were using before, but if that's fine, we can go with it.

File: src/utils/browser.ts (hunk 1 above)
Comment (sandrade-dcl) [reply src/utils/browser.ts]: I like it! Changed in https://github.com/decentraland/explorer-website/pull/34/commits/a3ce9a141f04a8070ee3adbbd9f3e6c502e3c495. Thanks!

**Comment by sandrade-dcl** [comment]
> There are some minor style changes, but there's one which I think that we should fix, the dialog padding between "Decentraland" and the "Play" button.
> 
> This should be easily fixable by using `margin-top` and `margin-bottom` with a margin similar of the one we have in production in the class that was added to color that text.
> 
> The text in general is slightly darker, as it's not the same one that you were using before, but if that's fine, we can go with it.

@LautaroPetaccio you're right, I didn't realize about this detail. I have added these margins to the `login-header-inmerse-dialog` class. Done in https://github.com/decentraland/explorer-website/pull/34/commits/1aa55cc0ff0320132b4e82c0cc92ece2d3d6b781. Thanks for the feedback!

**Review by RominaMarchetti — APPROVED** [review]

**Pull request merged**